lib/ccavenue-sdk.rb

```diff
@@ -363,11 +363,12 @@ def order_status_updated_at
     end
 
     ### merchant credentials validation
-    VALIDATION_SUCCESS_MSG = 'Providing Reference_No/Order No is mandatory'
+    VALIDATION_SUCCESS_MSGS = ['Providing Reference_No/Order No is mandatory',
```

Comment:
Not sure where this TODO was added, but please remember this is a public library that should work for vanilla spree. Any changes we make to this library should not rely on NEMO specific functionality (data-driven config, for example) and shouldn't reference JIRA tickets that don't make sense outside of our ecosystem.